### ISSUE TITLE
Error loading gzip decompress

### ISSUE THREAD

**Issue by djchou** [greemo/bberg-sftp#1]
I'm getting this error when importing the module.  

ImportError: cannot import name decompress

**Comment by greemo** [comment]
Yeah, sorry this is a python 3 library. Feel free to send me a pull request for making it backward compatible to python2, or I can make you a contributor
